### ISSUE TITLE
Failed to build iOS app - 'flutter_webview_plugin/FlutterWebviewPlugin.h' file not found

### ISSUE THREAD

**Issue by john-compass** [dart-flitter/flutter_webview_plugin#41]
Hi, I've recently integrated this plugin into my working Flutter project and attempted to build a preview for Android (which seems to work fine), and iOS which returned an error of the following:

Error output from Xcode build:
↳
    ** BUILD FAILED **
    
Xcode's output:
↳
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
    /flutterprojects/myapp/ios/Runner/GeneratedPluginRegistrant.m:6:9: fatal error: 'flutter_webview_plugin/FlutterWebviewPlugin.h' file not found
    #import <flutter_webview_plugin/FlutterWebviewPlugin.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.

I could use some advice to know if the error I am seeing is by design on my part.

**Comment by lejard-h** [comment]
@john-compass 
Hello, if you fixed the problem yourself, could you share the solution here for other people having the same problem ?

**Comment by brianmaxon** [comment]
The solution seems to be making sure that when you created the flutter project, you did *not* select the Swift iOS option. See this issue https://github.com/flutter/flutter/issues/15152 for details.

**Comment by Jleeowl** [comment]
Hey guys, sorry I did not reply sooner. Had to put this project on hold to work on other projects. The steps I took to resolve this issue had me inspecting the Runner.xcworkspace on xcode from the /ios directory of my flutter project. Under the Pods (build phases) I noticed that the header file for my plugin was under 'Projects' for some reason. By moving it to 'Public' the issue was resolved. At least for me. Hope that this will help others!

**Comment by john-compass** [comment]
Hey all, sorry I did not reply sooner. Had to put this project on hold to work on other projects. At the time I did not think to share, but, the steps I took to resolve this issue had me inspecting the Runner.xcworkspace on xcode from the /ios directory of my flutter project. 

Under the Pods (build phases) I noticed that the header file for my plugin was under 'Projects' for some reason. By moving it to 'Public' the issue was resolved, at least for me. 

I hope that what I share on here would be of help to others facing the same issue.

**Added note, I am also facing the same issue with my other plugins on the same project. As shown below, is how I fixed the issue for those plugins.

![fix](https://user-images.githubusercontent.com/5959650/37951921-b4650a72-31d0-11e8-9be2-f92f5bf2cbaf.png)
